curvefs/src/mds/topology/topology.cpp

```diff
@@ -885,6 +912,19 @@ std::vector<CopySetIdType> TopologyImpl::GetCopySetsInPool(
     return ret;
 }
 
+uint32_t TopologyImpl::GetCopySetNumInPool(
```

Comment:
GetCopySetsInPool function already exist and you can use directly.

Reply:
done